Code/Engine/Foundation/ezEngine.natvis

```diff
@@ -180,14 +180,14 @@
 			<Item Name="count">m_uiCount</Item>
 			<ArrayItems>
 				<Size>m_uiCount</Size>
-				<ValuePointer>m_ptr</ValuePointer>
+				<ValuePointer>m_pPtr</ValuePointer>
 			</ArrayItems>
 		</Expand>
 	</Type>
 
 	<!-- Basic Types -->
 	<Type Name="ezVariant">
-		<DisplayString Condition="m_Type == Type::TypedObject &amp;&amp; m_bIsShared">({m_Data.shared->m_pType->m_szTypeName,sb}*) {reinterpret_cast&lt;void*&gt;(m_Data.shared->m_Ptr)}</DisplayString>
+		<DisplayString Condition="m_Type == Type::TypedObject &amp;&amp; m_bIsShared">({m_Data.shared->m_pType->m_szTypeName,sb}*) {reinterpret_cast&lt;void*&gt;(m_Data.shared->m_pPtr)}</DisplayString>
```

Comment:
For the Variant shared data this is still called m_Ptr. @Ingrater is that something clang tidy missed? @Sanakan8472 Either revert the natvis change or fix it in ezVariant, either works for me.

Reply:
Natvis reverted as other changes are already in dev.